developers/bindings/thing-xml.md

```diff
@@ -488,7 +488,7 @@ In contrast to the properties defined in the 'ThingType' definitions the thing h
 ### Representation Property
 
 A thing type can contain a so-called `representation property`.
-This optional property contains the _name_ of a property whose value can be used to uniquely identify a device.
+This optional property contains the _**name**_ of a property whose value can be used to uniquely identify a device.
```

Comment:
You can remove the underlines if you want to replace the cursive text with a bold one.
```suggestion
This optional property contains the **name** of a property whose value can be used to uniquely identify a device.
```

Reply:
I think this doc update is great, because this is a topic that seems to be misunderstood by a lot of binding developers.
My unsolicited 2 cents is that I would also add the following two points in the developer docs.

First, if your binding makes use of a bridge, the representation property does not need to be globally unique, but only unique withing the context of the bridge, as long as in your discovery service you call ```DiscoveryResultBuilder.withBridge(bridgeUID)``` when building your DiscoveryResult. So if bridge A has child things with their representation properties set to 1, 2, and 3, and bridge B also has child things with their representation properties set to 1, 2, and 3, they will not conflict.

Second, if your thing has two configuration parameters (e.g. channel and address) where each individually is not unique, but the combination of the two **is** unique, you can do the following to make auto-ignore work properly in all cases:

1. Make up a new property (“id” in the examples below) and define that as the representation property in the child’s thing xml file.

Example:
```
<representation-property>id</representation-property>
```
2. In your discovery service, combine the two parameters into the new property and add it to the DiscoveryResult using DiscoveryResultBuilder. withProperties(). 

Example:
```
        Map<String, Object> properties = new HashMap<>();
        properties.put(PROPERTY_ADDRESS, address);
        properties.put(PROPERTY_CHANNEL, channel);
        properties.put(PROPERTY_ID, String.format("%d-%d", address, channel));
        DiscoveryResult result = DiscoveryResultBuilder.create(uid).withBridge(bridgeUID).withProperties(properties)
                .withRepresentationProperty(PROPERTY_ID).build();
```
3. In the child thing handler’s initialize() routine, again combine the two parameters into the same property in the same way, and call updateProperty() with it.

Example:
```
        String id = String.format("%d-%d", address, channel);
        updateProperty(PROPERTY_ID, id); // set representation property used by discovery
```

Reply:
Ok. It will be fixed in the next version.